### ISSUE TITLE
Optifine [1.12.2_HD_U_E3] Shaders Issue with Betweenlands when using Creative core mod

### ISSUE THREAD

**Issue by Elwood0** [sp614x/optifine#2479]
### **Overview:** 

So Optifine has a feature with it shaders that allow for them to be enabled or disabled for certain dimensions using an empty folder called "world[Dimension ID]" This has worked brilliantly for other mods such as the Betweenlands, which carries its own shader. _However_, This seems to have been broken when using Creative Core, even without any of its subsidiaries.

### **Issue:** 

Normally with the setup mentioned, when using a portal, the shader switches from what Optifine is currently using to the one from the Betweenlands.

However, when Creative core is running, when you go through a portal, the screen freezes. The game is still running and audio, movement, and full control still are taking place in the game, but you can't see it, only the last frame before going through the portal.

### **How to Reproduce:** 

**1.** Run the latest versions of Forge (14.23.5.2838), Optifine, (1.12.2_HD_U_E3), the Betweenlands (v3.4.7), and Creative core (v1.9.45_mc1.12.2).

**2.** Use one of the following Shaders I linked below. You could use anyone from the internet, but you would have to add the "world[Dimension ID]" yourself.

[Shaders]
(https://github.com/sp614x/optifine/files/3286573/Sildurs.Vibrant.Shaders.zip)

**3.** Launch the game and create a new world.

**4.** create a Betweenlands portal and attempt to go through.

**5.** Profit?

### **Conclusion:** 

If you have any questions about this, please let me know. I'm aware that this is a niche issue, but I hope it gets addressed soon. Thanks.

**Comment by kaporli** [comment]
how do i fix this issue

**Comment by NavanBethrax** [comment]
Same issue here i walked through the portal with enabled shaders and the screen just froze the game continues to run but i have to restart the client.

**Comment by rokybarua** [comment]
betweenlands is a mess of a mod

**Comment by rokybarua** [comment]
it causes mad issues with many different mods, optifine, twlight forest, witching gadgets to name a few

**Comment by EsotericRyoushi** [comment]
is there a solution yet?

**Comment by rokybarua** [comment]
yes, open the shader zip file and add a new empty file named world20 it will disable shaders for that specific dimension since there wont be any files inside that world file.
    On Friday, June 5, 2020, 01:41:21 p.m. EDT, Hunter Kern <notifications@github.com> wrote:  
 
 


is there a solution yet?

**Comment by EsotericRyoushi** [comment]
umm, that didn't fix it though? that's the whole thing that was the problem. Creating the folder doesn't fix it.

**Comment by rokybarua** [comment]
u have to create a folder inside the rar folder in shaders file and its case sensitive. That's how I fixed my problem. I made a empty folder in the shaderpacks file like an idiot. 
    On Saturday, June 6, 2020, 07:56:28 a.m. EDT, Hunter Kern <notifications@github.com> wrote:  
 
 


umm, that didn't fix it though? that's the whole thing that was the problem. Creating the folder doesn't fix it.

**Comment by danielbro9** [comment]
Its because the portal animation where it makes your screen swirl is part of its "shaders" effect. you have to disable the betweenlands shader in whatever dimension your portal to the betweenlands is in. trust me this will fix it.
